### PR TITLE
Feature/#3 review babel config and no fragment support

### PR DESCRIPTION
**Pull request by Nasdan**

### CODE REVIEW

File: 00_ES6/00_start/.babelrc [hunk 1]
```diff
@@ -1,11 +1,9 @@
 {
   "presets": [
-    "es2017",
-    "react",
-    "stage-0"
+    "env",
+    "react"
   ],
   "plugins": [
-    "transform-async-to-generator",
```
Comment (brauliodiez) [inline 00_ES6/00_start/.babelrc]: Which one is the setting to get async working es2017 and stage-0?

**Comment by brauliodiez** [comment]
About react.fragmen? can we use now <></> sintax? which one was the update on config to do that?

**Comment by Nasdan** [comment]
It looks like React.Fragment with syntax sugar it's only supported by Babel 7 (still in beta)

https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#babel.

See last comment: _Unfortunately, support for Babel 6.x is not available, and there are currently no plans to backport._

**Comment by brauliodiez** [comment]
@Nasdan Fragment... TypeScript 1 - Babel 0 :P

**Pull request merged**